pavement.py

```diff
@@ -267,9 +269,8 @@ def write_log_task(filename='Changelog'):
             stdout=subprocess.PIPE)
 
     out = st.communicate()[0].decode()
-    a = open(filename, 'w')
-    a.writelines(out)
-    a.close()
+    with open(filename, 'w') as a:
+        a.writelines(out)
```

Comment:
Can this be simplified to
```
    with open(filename, 'w') as log:
        st = subprocess.Popen(
            ['git', 'log',  '%s..%s' % (LOG_START, LOG_END)],	            ['git', 'log',  '%s..%s' % (LOG_START, LOG_END)],
            stdout=log)

Reply:
Maybe, but I'd rather not spend time to test that right now. This has worked for years and is only used by the release manager.